web_src/js/utils.js

```diff
@@ -26,13 +26,9 @@ export function isObject(obj) {
 
 // returns whether a dark theme is enabled
 export function isDarkTheme() {
-  if (document.documentElement.classList.contains('theme-auto')) {
-    return window.matchMedia('(prefers-color-scheme: dark)').matches;
-  }
-  if (document.documentElement.classList.contains('theme-arc-green')) {
-    return true;
-  }
-  return false;
+  const isDarkTheme = window.getComputedStyle(document.documentElement)
```

Comment:
I think this is a nice addition - as custom CSS themes which are made for gitea can now also specify that their theme is a dark theme via this way. 

But now we're removing the `theme-auto`, so a new media query in the CSS should be made to handle that part.
```css
@media (prefers-color-sheme: dark) {
 :root {
  --is-dark-theme: true;
 }
}
```

Reply:
By @silverwind's comment, this could be a nvm.

Reply:
Yes, the `auto` theme loads the entire `arc-green` theme including the `--is-dark-theme` variable if dark mode is enabled.

https://github.com/go-gitea/gitea/blob/main/web_src/less/themes/theme-auto.less

I have tested it and it works as expected.